ansible/veos

```diff
@@ -1,8 +1,8 @@
 [vm_host_1]
-STR-ACS-SERV-01 ansible_host=10.251.0.245
+STR-ACS-SERV-01 ansible_host=10.251.0.245 ansible_user=use_own_value ansible_become_password=use_own_value
```

Comment:
they are in vm_host group, and there is group_vars

```
johnar@1cafe0c9f994:/data/ansible$ more group_vars/vm_host/creds.yml 
---
ansible_user: use_own_value
ansible_password: use_own_value
ansible_sudo_pass: use_own_value
```

Reply:
I do not believe the change is needed.

Reply:
@lguohan Thanks for pointing out. I have updated PR with needed changes of using 
ansible_become_password instead of ansible_sudo_password.  
More details added in PR description. With this change it's working fine.